app.js

```diff
@@ -24,15 +27,15 @@ let env = nunjucks.configure(appViews, {
   watch: true // reload templates when they are changed. needs chokidar dependency to be installed
 })
 
-// make the function above available as a filter for all templates
+// make the function available as a filter for all templates
 env.addFilter('componentNameToMacroName', helperFunctions.componentNameToMacroName)
 
 // Set view engine
 app.set('view engine', 'njk')
 
 // Set up middleware to serve static assets
-app.use('/public', express.static(path.join(__dirname, '/public')))
-app.use('/icons', express.static(path.join(__dirname, '/public/icons')))
+app.use('/public', express.static(path.join(__dirname, configPaths.public)))
+app.use('/icons', express.static(path.join(__dirname, configPaths.public + 'icons')))
```

Comment:
Can we rely on path.join to join `configPaths.public` and `'icons'`?

i.e.

```
path.join(__dirname, configPaths.public, 'icons')
```

rather than

```
path.join(__dirname, configPaths.public + 'icons'))
```